client/state/sites/hooks/use-site-global-styles-status.ts

```diff
@@ -33,13 +68,38 @@ export const getGlobalStylesInfoForSite = ( siteId: number | null ): GlobalStyle
 		.catch( () => DEFAULT_GLOBAL_STYLES_INFO );
 };
 
-export function useSiteGlobalStylesStatus( siteId: number ): GlobalStylesStatus {
-	const { data } = useQuery( {
-		queryKey: [ 'globalStylesInfo', siteId ],
-		queryFn: () => getGlobalStylesInfoForSite( siteId ),
+export function useSiteGlobalStylesStatus(
```

Comment:
What do you think if we rename this to `useSiteCustomStylesStatus` instead since we seem to be adopting the `Custom Syle/s` naming across our code now?

Reply:
I think that'll make things more confusing because the data returned is `globalStylesInUse`, `shouldLimitGlobalStyles`, `globalStylesInPersonalPlan`. Also, Core uses refers to the feature internally as "Global Styles" too and it's exposed to users as "(Custom) Styles".

Reply:
We will have to accept the existing semantic disparity between the domain and the code, even though it is not an ideal situation.